### PR TITLE
fix documentation and scripts for developers 

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Few fixes for update-code-for-all-services script:

- Datamodel is stored in .../datamodel/... instead of .../resources/...
- Add missing pcn-iptables
- Fix path for transparent-helloworld

### CODE REVIEW

File: Documentation/devel/update-code-for-all-services.sh [hunk 1]
```diff
@@ -15,12 +16,13 @@ SERVICES=(#bridge
           pbforwarder
           router
           simplebridge
-          simpleforwarder)
+          simpleforwarder
+          transparent-helloworld)
```
Comment (mbertrone) [inline Documentation/devel/update-code-for-all-services.sh]: Should we exit this script if no `POLYCUBE_HOME` is set?

```suggestion

if [ -z "$POLYCUBE_HOME" ]; then
    echo "POLYCUBE_HOME is not set"
    exit 1
fi
```

Comment (mauriciovasquezbernal) [reply Documentation/devel/update-code-for-all-services.sh]: I added it, thanks!

**Comment by mauriciovasquezbernal** [comment]
I removed the devel folder, scripts were moved to scripts folder and the documentation file to `Documentation/codegen.rst`.

Please notice that I only updated the paths to the files, the content of the documentation itself should be updated in few days.

**Comment by mbertrone** [comment]
LGTM! Thanks Mauricio

**Pull request merged**